aws/resources/ec2_vpc.go

```diff
@@ -57,14 +57,21 @@ func (v *EC2VPCs) getFirstSeenTag(vpc ec2.Vpc) (*time.Time, error) {
 	return nil, nil
 }
 
-func (v *EC2VPCs) getAll(c context.Context, configObj config.Config) ([]*string, error) {
+func (v *EC2VPCs) getAll(_ context.Context, configObj config.Config) ([]*string, error) {
+	// Note: This filter initially handles non-default resources and can be overridden by passing the only-default filter to choose default VPCs.
+	var isdefault = "false"
```

Comment:
make sense to use boolean variables? if left as string, in the future(during refactoring maybe) someone may try to put different value than true/false

Reply:
Hey @denis256, this is the value we pass in as part of the `DescribeVpcs` endpoint. 
We use boolean inside the configObj for this. Essentailly you need to set either `true` or `false` as a string. That's how the API is defined.

Reply:
Yes, I was thinking of representing value before passing it to AWS, and don't allow invalid value to be set in:

```
var isdefault := false
...
Values: awsgo.StringSlice([]string{fmt.Sprintf("%v", isdefault})
...
...
```

`isdefault` can take only valid values, and will be transformed in string when passed to AWS